pallets/crowdloan-claim/src/lib.rs

```diff
@@ -751,6 +756,22 @@ impl<T: Config> Pallet<T> {
 		T::PalletId::get().into_account()
 	}
 
+	/// Build a sorted hash of two given hash values.
+	///
+	/// Hash a:b if a < b, else b:a. Uses the runtime module's hasher.
+	pub fn sorted_hash_of(a: &T::Hash, b: &T::Hash) -> T::Hash {
```

Comment:
We should use proofs for verifying the merkle proofs. Here is a mock on how to use it https://github.com/centrifuge/centrifuge-chain/blob/90fca013acaa449a887afcd5763cdfac7dc886e2/libs/proofs/src/mock.rs#L15

Also, blake256 is our  hashing function for our merkle trees. While its the same for T::Hashing, it would make it easier for us to use proofs library where the hash function is in our control. 
We would need to change it claims as well at some point

Reply:
But we would need to remove the `Hasher` trait of the lib here and use the rust specific definition. Otherwise, we would need to manually implement `Hasher` for blake256. 
WDYT?